src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.cpp

```diff
@@ -681,6 +681,28 @@ bool hasNonIdentityLayout(ValueRange operands) {
   return false;
 }
 
+//===----------------------------------------------------------------------===//
+// Support functions for parallel region.
+//===----------------------------------------------------------------------===//
+
+// Return the outermost loop within (firstDim, lastDim] for which (lb-Ub) >
```

Comment:
I guess you mean `(ub - lb)` not `(lb-Ub)`, also `[firstDim, lastDim)` not `(firstDim, lastDim]`.
Please update these in the header file also if so.

Reply:
Thanks, in Europe, we use `[num-inclusive, num-exclusive[` notation which I find much clearer... Wonder what it is in Japan. Thanks for catching the mistake.